ddtrace/contrib/pytest_bdd/__init__.py

```diff
@@ -32,3 +32,12 @@
         _default_service="pytest_bdd",
     ),
 )
+
+
+def get_version():
```

Comment:
I'd say this should use the same logic as https://github.com/DataDog/dd-trace-py/blame/2a1f4cab08b326a7b97ff4e0515da9db093f3dea/ddtrace/contrib/pytest_bdd/plugin.py#L81-L86 , but it should actually probably use the same code.